scripts/TCL/positionData/198350_2022-02-28.json

```diff
@@ -0,0 +1,8 @@
+{
+    "amount0": 57.70800678,
```

Comment:
can this be known in advance? guess this depends on the actual exec tx, no?

Reply:
that's true, it was more on the purpose of bringing this file into the repo and then update it with precise figure, since we minted on the 28th but have not push this json file